[en]: Update intro/permission and fix typo (#5610)
DaoCloud/DaoCloud-docs: docs/en/docs/insight/quickstart/res-plan/prometheus-res.md

---

> MTPE:WANG0608GitHub
> Date:2024-09-24

Reviewer: ```suggestion
MTPE: WANG0608GitHub
Date: 2024-09-24
```

---

# Prometheus Resource Planning

In the actual use of Prometheus, affected by the number of cluster containers and the opening of Istio,
the CPU, memory and other resource usage of Prometheus will exceed the set resources.

In order to ensure the normal operation of Prometheus in clusters of different sizes,
it is necessary to adjust the resources of Prometheus according to the actual size of the cluster.

## Reference resource planning

In the case that the mesh is not enabled, the test statistics show that the relationship

> between the system Job index and pods is **Series number = 800\*pods number**

Reviewer: ```suggestion
between the system Job index and pods is **Series  count = 800 \* pod count**
```

When the service mesh is enabled, the magnitude of the Istio-related metrics generated

> by the pod after the feature is enabled is **Series number = 768\*pods number**

Reviewer: ```suggestion
by the pod after the feature is enabled is **Series count = 768 \* pod count**
```

### When the service mesh is not enabled

The following resource planning is recommended by Prometheus when **the service mesh is not enabled** :

> | Cluster size (pods number) | Metrics (service mesh is not enabled) | CPU (core) | Memory (GB) |

Reviewer: ```suggestion
| Cluster size (pod count) | Metrics (service mesh is not enabled) | CPU (core) | Memory (GB) |
```

The following resource planning is recommended by Prometheus in the scenario of **starting the service mesh**:

> | Cluster size (pods number) | metric volume (service mesh enabled) | CPU (core) | Memory (GB) |

Reviewer: ```suggestion
| Cluster size (pod count) | metric volume (service mesh enabled) | CPU (core) | Memory (GB) |
```
